pkg/apic/resourcePagination.go

```diff
@@ -0,0 +1,62 @@
+package apic
+
+import (
+	"encoding/json"
+	"strconv"
+
+	coreapi "github.com/Axway/agent-sdk/pkg/api"
+	"github.com/Axway/agent-sdk/pkg/apic/apiserver/clients/definitions/v1alpha1"
+	"github.com/Axway/agent-sdk/pkg/apic/apiserver/models/management/v1alpha1"
+	"github.com/Axway/agent-sdk/pkg/util/log"
+)
+
+func (c *ServiceClient) fubar() error {
+	query := map[string]string{
+		"query": "metadata.references.name==" + "scrum",
+		"sort":  "metadata.audit.createTimestamp,DESC",
+	}
+	resources, err := c.getAPIResources(query, c.cfg.GetInstancesURL(), "")
+	if err != nil {
+		return err
+	}
+
+	resources = resources.([]v1alpha1.APIServiceInstance)
+}
+
+func (c *ServiceClient) getAPIResources(queryParams map[string]string, URL, stage string) ([]interface{}, error) {
+	morePages := true
+	page := 1
+
+	resourceInstance := make([]interface{}, 0)
+
+	for morePages {
+		query := map[string]string{
+			"page":     strconv.Itoa(page),
+			"pageSize": strconv.Itoa(apiServerPageSize),
```

Comment:
might there ever be cases where you want the pagesize for different resources to be different? If so maybe pass it in rather than having a single global value?

Reply:
see above... check in was a DRAFT holder... Sorry for the confusion